docs/_docs/cygwin.md

```diff
@@ -35,17 +35,18 @@ From an elevated cmd.exe prompt set the GOPATH variable in Windows and add the `
 
 Then `go get -u -v` the following packages:
 
-    go get -u -v github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
-    go get -u -v github.com/grpc-ecosystem/grpc-gateway/protoc-gen-openapiv2
-    go get -u -v github.com/golang/protobuf/protoc-gen-go
+    go get -u -v github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway
+    go get -u -v github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2
+    go get -u -v google.golang.org/protobuf/cmd/protoc-gen-go
+    go get -u -v google.golang.org/grpc/cmd/protoc-gen-go-grpc
```

Comment:
We should drop the `-u -v` as well, this is not how things are installed anymore.